projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -1039,3 +1039,50 @@ mail:
     ssl:
       protocols: "TLSv1.2"
     port: 587
+
+# Alert manager email notification configuration for the Data-Pipelines/Data Jobs
+# (dp) k8s deployment.
+# More information on individual fileds can be found here:
+# https://docs.openshift.com/container-platform/4.10/rest_api/monitoring_apis/alertmanagerconfig-monitoring-coreos-com-v1alpha1.html#specification
+# Pre-requisites for the alertmanager is that prometheus-operator is installed:
+# helm install prometheus-operator prometheus-community/kube-prometheus-stack
+alertmanager:
+  enabled: false
+  # Authentication credentials are mounted into a k8s secret, that's why they are separate.
+  authentication:
+    secretName: "alertmanager-authentication-secret"
+    authPassword: ""
+    authSecret: ""
+    authSecretKey: ""
+    authIdentity: ""
+  metadata:
+    labels:
+      role: infra
+    name: dp-notifications
+    namespace: default
+  spec:
+    receivers:
+      - emailConfigs:
+          - from: ""
+            headers:
+              - key: subject
+                value: ""
+            hello: localhost
```

Comment:
what is this hello: localhost here for?

Reply:
https://docs.openshift.com/container-platform/4.10/rest_api/monitoring_apis/alertmanagerconfig-monitoring-coreos-com-v1alpha1.html#spec-receivers-emailconfigs-2

The hostname to identify to the SMTP server.